cybergis/general/summa.py

```diff
@@ -28,6 +28,27 @@ def __init__(self, walltime, nodes, jobname,
         self.simg_path = self.simg_path
 
 
+class SummaCometSBatchScript(KeelingSBatchScript):
+
+    name = "SummaCometSBatchScript"
+    SCRIPT_TEMPLATE = \
+'''#!/bin/bash
+#SBATCH --job-name=$jobname
+#SBATCH --nodes=$nodes
+module load singularity/2.6.1
+sbatch -t $walltime singularity exec $simg_path python $userscript_path'''
+
+    simg_path = "/home/cybergis/SUMMA_IMAGE/pysumma_ensemble.img"
+    userscript_path = None
+
+    def __init__(self, walltime, nodes, jobname,
```

Comment:
@CarnivalBug same copy and paste issue for this __init__()?

Reply:
we can't remove init here.